nncf/experimental/post_training/algorithms/quantization/fast_bias_correction.py

```diff
@@ -0,0 +1,195 @@
+from nncf.experimental.onnx.graph.transformations.commands import ONNXTargetPoint
+from nncf.experimental.onnx.tensor import ONNXNNCFTensor
+from nncf.experimental.post_training.algorithms import AlgorithmParameters
+from nncf.experimental.post_training.algorithms.algorithm import Algorithm
+from nncf.experimental.post_training.algorithms.algorithm import PostTrainingAlgorithms
+from nncf.experimental.post_training.graph.factories import NNCFGraphFactory
+from nncf.experimental.post_training.graph.factories import PTQBiasCorrectionCommandFactory
+from nncf.experimental.post_training.graph.factories import PTQTargetPointFactory
+from nncf.experimental.post_training.graph.factories import PTQTransformationLayoutFactory
+from nncf.experimental.post_training.graph.factories import PTQMeanStatisticCollectorFactory
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPoint
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPointsContainer
+
+ModelType = TypeVar('ModelType')
+
+
+class FastBiasCorrectionParameters(AlgorithmParameters):
+    """
+    Base class of FastBiasCorrection parameters.
+    """
+
+    def __init__(self, number_samples: int = 100):
+        self.number_samples = number_samples
+
+    def to_json(self) -> Dict[str, Union[str, float, int]]:
+        """
+        Serialize all FastBiasCorrection parameters to JSON.
+        """
+
+
+class FastBiasCorrection(Algorithm):
+
+    def __init__(self, parameters: FastBiasCorrectionParameters):
+        super().__init__()
+        self.number_samples = parameters.number_samples
+        self.nncf_graph = None
+        self._target_points_to_correct = []  # type: List[ONNXTargetPoint]
+        self._axes = (0, 2, 3)
+
+    def generate_stat_collector(self, model_backend) -> TensorStatisticCollectorBase:
+        return PTQMeanStatisticCollectorFactory.create(
+            model_backend,
+            reduction_shape=self._axes,
+            num_samples=self.number_samples)
+
+    @staticmethod
+    def _input_filter_func(point):
+        return PostTrainingAlgorithms.FastBiasCorrection in point.algorithm_to_tensor_collectors and \
+            point.target_point.type == TargetType.PRE_LAYER_OPERATION
+
+    @staticmethod
+    def _output_filter_func(point):
+        return PostTrainingAlgorithms.FastBiasCorrection in point.algorithm_to_tensor_collectors and \
+            point.target_point.type == TargetType.POST_LAYER_OPERATION
+
+    def _apply(self, model: onnx.ModelProto, engine: ONNXEngine,
+               statistic_points: StatisticPointsContainer) -> onnx.ModelProto:
+
+        model_backend = infer_backend_from_model(model)
+
+        transformation_layout = PTQTransformationLayoutFactory.create(model_backend)
+        transformation_commands = []
+        nncf_graph = NNCFGraphFactory.create(model)
+
+        biased_nodes = self.get_nodes_to_correct(model)
```

Comment:
```suggestion
        biased_nodes = nncf_graph.get_nodes_by_metatypes(LAYERS_WITH_BIAS_METATYPES)
```

I think it is better to avoid redundant methods.

Reply:
Done.